addons/mail/static/src/utils/common/misc.js

```diff
@@ -191,3 +192,45 @@ export function convertToEmbedURL(url) {
     }
     return { url: null, provider: null };
 }
+
+/**
+ * Compute the preview of the message containing attachments that should shown for the thread.
+ *
+ * @param {object} messages
+ * @param {object} thread
+ * @returns {string}
+ */
+
+export function attachmentMessagePreview(messages = undefined, thread = undefined) {
```

Comment:
Why do we have this a a misc utility?
If anything this should be a function on message.